### ISSUE TITLE
Records Commit Example Shows a Data Field But Description Does Not Clarify The Field

### ISSUE THREAD

**Issue by andorsk** [decentralized-identity/decentralized-web-node#207]
On https://identity.foundation/decentralized-web-node/spec/#recordscommit, request for further clarity on the `data` field within a records commit. Shown in the example, but not expanded upon in the description section. 

Example Given:

```json
{ // Message
  "data": {...},
  "recordId": "b65b7r8n7bewv5w6eb7r8n7t78yj7hbevsv567n8r77bv65b7e6vwvd67b6",
  "descriptor": { // Message Descriptor
    "method": "RecordsCommit",
    "dataCid": CID(data),
    "parentId": CID(ANCESTOR_CID),
    "dateCreated": 123456789,
    "commitStrategy": "json-merge",
    "dataFormat": DATA_FORMAT
  }
}
```

Description:

> RecordsCommit messages are JSON objects that include general [Message Descriptor](https://identity.foundation/decentralized-web-node/spec/#message-descriptors) properties and the following additional properties, which MUST be composed as follows:
> 
> The message object MUST contain a recordId property, and its value MUST be the recordId of the logical record the entry corresponds with.
> If the message object is attached to a Protocol, and its value MUST be a [Computed Context ID](https://identity.foundation/decentralized-web-node/spec/#computed-context-ids). If the message is not attached to a Protocol, it MUST NOT contain a contextId property.
> The message object MUST contain a descriptor property, and its value MUST be a JSON object composed as follows:
> The object MUST contain a method property, and its value MUST be the string RecordsCommit.
> The object MUST include a parentId property, and its value MUST be the stringified [Version 1 CID](https://docs.ipfs.io/concepts/content-addressing/#identifier-formats) of the [DAG CBOR](https://github.com/ipld/specs/blob/master/block-layer/codecs/dag-cbor.md) encoded descriptor object of the previous RecordsWrite or RecordsCommit ancestor in the record’s lineage.
> The object MUST contain a commitStrategy property, and if present its value MUST be a string from the table of registered [Commit Strategies](https://identity.foundation/decentralized-web-node/spec/#commit-strategies).
> The object MUST contain a dateCreated property, and its value MUST be an [[RFC3339](https://identity.foundation/decentralized-web-node/spec/#ref:RFC3339)] ISO 8601 timestamp that MUST be set and interpreted as the time the commit was generated.

**Comment by andorsk** [comment]
Discussed on DIF call: 
- Remove the embedded data property on the spec. 
- Add a section that defines the data CID property as the way to represent data. See `Transport` section for how to represent data in the transport.

**Comment by andorsk** [comment]
that works. Thanks @csuwildcat !

**Comment by andorsk** [comment]
@csuwildcat Was this merged? Can this be closed?

**Comment by csuwildcat** [comment]
Yes, it was merged, but I'd leave it open because it's not a full year resolved general issue
